### PR TITLE
boa: first tf demo for boa.

### PR DESCRIPTION
**Pull request by Txiaozhe**

### CODE REVIEW

**Comment by Txiaozhe** [comment]
boa needs HelloWorld~

File: example/boa/tf2/first-tensorflow.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict'
```
Comment (yorkie) [inline example/boa/tf2/first-tensorflow.js:1]: rename to hello.js? tensorflow might be duplicated with tf2.

Comment (Txiaozhe) [reply example/boa/tf2/first-tensorflow.js]: renamed

Comment (Txiaozhe) [reply example/boa/tf2/first-tensorflow.js]: I plan to add some boa demo for python and tensorflow, and have you some todo list？

Comment (yorkie) [reply example/boa/tf2/first-tensorflow.js]: Python?

Comment (Txiaozhe) [reply example/boa/tf2/first-tensorflow.js]: Depending on your develop plan~

**Pull request merged**